src/Common/src/Interop/Windows/mincore/Interop.GetComputerName.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [DllImport(Libraries.Kernel32_L2, CharSet = System.Runtime.InteropServices.CharSet.Unicode, EntryPoint = "GetComputerNameW")]
+        private extern static int GetComputerName(char[] lpBuffer, ref uint nSize);
+
+        internal static string GetComputerName()
+        {
+            char[] buffer = new char[256];
```

Comment:
This was an existing code in the Interop.Windows.cs file I deleted, I simply moved it around :)

I believe we are using MAX_MACHINENAME_LENGTH for the buffer-size. I agree with you that using MAX_COMPUTERNAME_LENGTH + 1 as the buffer size should suffice but this as-is should cause no harm. I will take a closer look at this separately to see if we can update this to GetComputerNameEx instead. Thanks!